run_md.py

```diff
@@ -0,0 +1,879 @@
+                    shutil.copy(protein, os.path.join(wdir_protein, os.path.basename(protein)))
+                if not os.path.isfile(os.path.join(wdir_protein, 'topol.top')):
+                    shutil.copy(topol, os.path.join(wdir_protein, 'topol.top'))
+                # multiple chains
+                for posre_protein in posre_list_protein:
+                    if not os.path.isfile(os.path.join(wdir_protein, os.path.basename(posre_protein))):
+                            shutil.copy(posre_protein, os.path.join(wdir_protein, os.path.basename(posre_protein)))
+                if topol_itp_list is not None:
+                    for topol_itp in topol_itp_list:
+                        if not os.path.isfile(os.path.join(wdir_protein, os.path.basename(topol_itp))):
+                            shutil.copy(topol_itp, os.path.join(wdir_protein, os.path.basename(topol_itp)))
+
+
+        else:
+            logging.warning(f'{os.path.join(wdir_protein, pname)}.gro and topol.top files exist. '
+                            f'Protein preparation step will be skipped.')
+
+
+        # Part 1. Preparation. Run on each cpu
+        dask_client = init_dask_cluster(hostfile=hostfile, n_tasks_per_node=ncpu, ncpu=ncpu)
+        try:
+            system_lig_data = [] # wdir_ligand_cur, molid, resid
+            if system_lfile is not None:
+                if not os.path.isfile(system_lfile):
+                    raise FileExistsError(f'{system_lfile} does not exist')
+
+                mols = supply_mols(system_lfile, set_resid=None)
+                for res in calc_dask(prep_ligand, mols, dask_client,
+                                                  script_path=script_path, project_dir=project_dir,
+                                                  wdir_ligand=wdir_system_ligand,
+                                                  addH=addH):
+                    if not res:
+                        logging.error(f'Error with system ligand (cofactor) preparation. The calculation will be interrupted\n')
+                        return
+                    system_lig_data.append(res)
+
+            var_lig_data = [] # wdir_ligand_cur, molid, resid
+            if lfile is not None:
+                if not os.path.isfile(lfile):
+                    raise FileExistsError(f'{lfile} does not exist')
+
+                mols = supply_mols(lfile, set_resid='UNL')
+                for res in calc_dask(prep_ligand, mols, dask_client,
+                                      script_path=script_path, project_dir=project_dir,
+                                      wdir_ligand=wdir_ligand, addH=addH):
+                    if res:
+                        var_lig_data.append(res)
+
+            # make all.itp and create complex
+            var_complex_prepared_dirs = []
+            if var_lig_data:
+                for res in calc_dask(run_complex_prep, var_lig_data, dask_client,
```

Comment:
check whether it is possible to use the same call for both case, try to use `[[]]` in the second case
if it will work, add a line before
```python
if not var_lig_data:
    var_lig_data = [[]]
```